src/controllers/controller.radar.js

```diff
@@ -136,7 +139,7 @@ module.exports = function(Chart) {
 
 				// Now pivot the point for animation
 				point.pivot();
-			}, this);
+			}, me);
```

Comment:
`me` is not needed here because you can directly use it in function closure but also because `this` is not referenced in the each() loop. Then I would not cache `this` since there are only 2 occurrences.